apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -149,6 +159,10 @@ impl PlannerMode {
             QueryPlannerMode::New | QueryPlannerMode::Both => {
                 Ok(Some(Self::rust(schema, configuration)?))
             }
+            QueryPlannerMode::BothBestEffort => match Self::rust(schema, configuration) {
+                Ok(planner) => Ok(Some(planner)),
+                Err(_) => Ok(None),
+            },
```

Comment:
This makes fallback completely silent. Should we log a warning instead?

Reply:
it'd be nice to warn for fed 1 (since it's a future compatibility issue), but maybe not for Rust QP bugs or missing features

Reply:
It seems unlike that constructing the Rust QP would fail in places that the JS QP doesn't. If we were to log errors here, what would we want to log?

Reply:
Added a `tracing::warn!` call and a test for it